lib/ansible/modules/system/java_keystore.py

```diff
@@ -186,7 +191,7 @@ def cert_changed(module, openssl_bin, keytool_bin, keystore_path, keystore_pass,
         os.remove(certificate_path)
 
 
-def create_jks(module, name, openssl_bin, keytool_bin, keystore_path, password):
+def create_jks(module, name, openssl_bin, keytool_bin, keystore_path, password, in_password):
```

Comment:
In [Java-land](https://docs.oracle.com/javase/7/docs/technotes/tools/windows/keytool.html) `password` would be known as `storepass`, the password to access the actual keystore. `in_password` would be called `keypass` which is the pass for the individual cert _inside_ they keystore. Just in case you wanted to follow Java convention for this addition